installer_linux/make_deb.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
`set -e`?

Reply:
Oh just saw this now. Yeah set -e would be good. Let me let this force push run first.